content/docs/2.5/concepts/authentication.md

```diff
@@ -92,28 +92,28 @@ metadata:
   namespace: default # must be same namespace as the ScaledObject
 spec:
   podIdentity:
-      provider: none | azure | aws-eks | aws-kiam  # Optional. Default: none
-  secretTargetRef:                                    # Optional.
-  - parameter: {scaledObject-parameter-name}          # Required.
-    name: {secret-name}                               # Required.
-    key: {secret-key-name}                            # Required.
-  env:                                                # Optional.
-  - parameter: {scaledObject-parameter-name}          # Required.
-    name: {env-name}                                  # Required.
-    containerName: {container-name}                   # Optional. Default: scaleTargetRef.envSourceContainerName of ScaledObject
-  hashiCorpVault:                                     # Optional.
-    address: {hashicorp-vault-address}                # Required.
-    namespace: {hashicorp-vault-namespace}            # Optional. Default is root namespace. Useful for Vault Enterprise
-    authentication: token | kubernetes                # Required.
-    role: {hashicorp-vault-role}                      # Optional.
-    mount: {hashicorp-vault-mount}                    # Optional.
-    credential:                                       # Optional.
-      token: {hashicorp-vault-token}                  # Optional.
-      serviceAccount: {path-to-service-account-file}  # Optional.
-    secrets:                                          # Required.
-    - parameter: {scaledObject-parameter-name}        # Required.
-      key: {hasicorp-vault-secret-key-name}           # Required.
-      path: {hasicorp-vault-secret-path}              # Required.
+      provider: none | azure | gcp | spiffe | aws-eks | aws-kiam  # Optional. Default: none
```

Comment:
`spiffe` ?

Reply:
Saw that in the code (https://github.com/kedacore/keda/blob/a223a0ee2a3ad0a5df8ca73308c41116a30af51e/apis/keda/v1alpha1/triggerauthentication_types.go#L95-L100) and assumed all these types work.

Reply:
@tomkerkhove PTAL

Reply:
AFAIK we don't implement this, this part of the code is just a leftover I'd say. We should have remove it from the codebase or implement.